src/main/java/org/broadinstitute/hellbender/tools/exome/HashTargetAnnotationCollection.java

```diff
@@ -0,0 +1,66 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.*;
+
+/**
+ * Simple target annotation collection based on a hash map from the annotation to its value as an string.
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+final class HashTargetAnnotationCollection implements TargetAnnotationCollection {
+
+    private final Map<TargetAnnotation, String> values;
```

Comment:
Since `TargetAnnotation` is an `Enum` couldn't this be implemented as an `EnumMap`?